python/paddle/tensor/math.py

```diff
@@ -7871,3 +7871,176 @@ def isreal(x, name=None):
+
+            >>> # Set `assume_unique` to True only when `elements` and `test_elements` contain unique values, otherwise the result may be incorrect.
+            >>> elements = paddle.to_tensor([0., 1., 2.]*20).reshape([20, 3])
+            >>> test_elements = paddle.to_tensor([0., 1.]*20)
+            >>> correct_result = paddle.isin(elements, test_elements, assume_unique=False)
+            >>> print(correct_result)
+            Tensor(shape=[20, 3], dtype=bool, place=Place(cpu), stop_gradient=True,
+            [[True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False],
+             [True , True , False]])
+
+            >>> incorrect_result = paddle.isin(elements, test_elements, assume_unique=True)
+            >>> print(incorrect_result)
+            Tensor(shape=[20, 3], dtype=bool, place=Place(gpu:0), stop_gradient=True,
+            [[True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , True ],
+             [True , True , False]])
+
+    """
+    if not isinstance(elements, (paddle.Tensor, Variable, paddle.pir.Value)):
+        raise TypeError(f"x must be tensor type, but got {type(elements)}")
+    if not isinstance(
+        test_elements, (paddle.Tensor, Variable, paddle.pir.Value)
+    ):
+        raise TypeError(f"x must be tensor type, but got {type(test_elements)}")
+
+    check_variable_and_dtype(
+        elements,
+        "elements",
+        [
+            'float32',
+            'float64',
+            'int32',
+            'int64',
+        ],
+        "isin",
+    )
+
+    check_variable_and_dtype(
+        test_elements,
+        "test_elements",
+        [
+            'float32',
+            'float64',
+            'int32',
+            'int64',
```

Comment:
same date type issue as above

Reply:
Temporarily not since `paddle.searchsorted` only support 'float32', 'float64', 'int32' and 'int64'.